common/error.h

```diff
@@ -138,35 +138,36 @@ class [[nodiscard]] ErrorOr {
 class ErrorBuilder {
  public:
   explicit ErrorBuilder(std::string location = "")
-      : location_(std::move(location)) {}
+      : location_(std::move(location)),
+        out_(std::make_unique<RawStringOstream>()) {}
 
```

Comment:
I take it the issue was that this class needs to be movable? If so, it'd be a nice drive-by fix to record that (per the [style guide](https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types)):

```suggestion

  ErrorBuilder(ErrorBuilder&&) = default;
  ErrorBuilder& operator=(ErrorBuilder&&) = default;

```